Mathlib/RingTheory/Invariant.lean

```diff
@@ -26,6 +29,15 @@ Let `G` be a finite group acting on a commutative ring `B` satisfying `Algebra.I
 * `IsFractionRing.stabilizerHom_surjective`: if `Q` is a prime ideal of `B` lying over a prime
   ideal `P` of `A`, then the stabilizer subgroup of `Q` surjects onto `Aut(Frac(B/Q)/Frac(A/P))`.
 
+Let `G` be a profinite group acting continuously on a commutative ring `B` satisfying
```

Comment:
What do you think about having `RingTheory/Invariant/Basic.lean` containing the results for finite groups and `RingTheory/Invariant/Profinite.lean` containing the results for profinite groups? Or is it actually unlikely that someone will need to finite group version but not the profinite group version?